tests/test_gpu_common.py

```diff
@@ -1140,3 +1140,54 @@ def test_empty_arrays(self):
         op(time_m=0, time_M=9)
         sf.manual_gather()
         assert np.all(f.data == 1.)
+
+    @skipif('device-openmp')
+    @pytest.mark.parallel(mode=4)
+    def test_degenerate_subdomainset(self):
+        """
+        MFE for issue #1766
```

Comment:
Add issue number in test name as in the format:
`def test_issue_#issue()`
issued in a few cases:

such as issue_1592, issue_1725, issue_1761, issue_1627

?

Reply:
yeah, we loosely following that convention, unfortunately... I did it as in the test above, actually!